lib/src/code_controller.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:math';
```

Comment:
Added redundant types.
We will be pulling changes from BertrandBev, so this file should not be edited without need.
Will not merge this.